src/cmd/dut.rs

```diff
@@ -241,6 +241,26 @@ type DutAction = Box<fn(&SshInfo) -> Result<()>>;
 fn do_reboot(s: &SshInfo) -> Result<()> {
     s.run_cmd_piped(&["reboot; exit"])
 }
+fn do_reboot_other(s: &SshInfo) -> Result<()> {
+    let rootdev = s.get_rootdev()?;
+    let rootdisk = s.get_rootdisk()?;
+    let kern_a = s.get_partnum_kern_a()?;
+    let kern_b = s.get_partnum_kern_b()?;
+    let root_a = s.get_partnum_root_a()?;
+    let root_b = s.get_partnum_root_b()?;
+    let (other_name, other_kern, other_root) = if rootdev.ends_with(&root_a) {
+        ("B", &kern_b, &root_b)
+    } else if rootdev.ends_with(&root_b) {
+        ("A", &kern_a, &root_a)
+    } else {
+        return Err(anyhow!("unsupported partition layout"));
+    };
+    println!("rebooting into {other_name} ({rootdisk}p{other_root})");
+    let cmd = format!("cgpt prioritize -P2 -i {other_kern} {rootdisk}");
+    s.run_cmd_stdio(&cmd)?;
+
+    do_reboot(s)
```

Comment:
I think it is a good idea to separate this command into two:
```
toggle_boot_partition reboot
```

Reply:
removed the reboot part